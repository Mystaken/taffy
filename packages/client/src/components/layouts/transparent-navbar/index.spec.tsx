import React from 'react';
import { create } from 'react-test-renderer';
import { TransparentNavbar } from '.';

describe('<TransparentNavbar />', () => {
  it('has rendered correctly', () => {
    const component = create(<TransparentNavbar />);
    const tree = component.toJSON()!;
    expect(tree).toMatchSnapshot();
  });
});
