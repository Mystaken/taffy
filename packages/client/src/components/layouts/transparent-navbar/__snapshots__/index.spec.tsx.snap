// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<TransparentNavbar /> has rendered correctly 1`] = `
<div
  className="MuiGrid-root makeStyles-navbar-1 MuiGrid-container MuiGrid-direction-xs-column MuiGrid-justify-xs-center"
>
  <div
    className="MuiGrid-root MuiGrid-container MuiGrid-align-items-xs-center MuiGrid-justify-xs-space-between"
  >
    <div
      className="MuiGrid-root MuiGrid-item"
    >
      <div
        aria-disabled={false}
        className="MuiButtonBase-root MuiListItem-root MuiListItem-gutters MuiListItem-button"
        onBlur={[Function]}
        onDragLeave={[Function]}
        onFocus={[Function]}
        onKeyDown={[Function]}
        onKeyUp={[Function]}
        onMouseDown={[Function]}
        onMouseLeave={[Function]}
        onMouseUp={[Function]}
        onTouchEnd={[Function]}
        onTouchMove={[Function]}
        onTouchStart={[Function]}
        role="button"
        tabIndex={0}
      >
        <div
          className="MuiListItemAvatar-root"
        >
          <div
            className="MuiAvatar-root MuiAvatar-circle"
          >
            <img
              alt="Taffy"
              className="MuiAvatar-img"
              src="/static/icons/illustrious_comics_logo.png"
            />
          </div>
        </div>
        <h5
          className="MuiTypography-root MuiTypography-h5 MuiTypography-colorTextSecondary"
        >
          Taffy
        </h5>
      </div>
    </div>
    <div
      className="MuiGrid-root MuiGrid-item"
    />
  </div>
</div>
`;
